### ISSUE TITLE
free space handling

### ISSUE THREAD

**Issue by jospoortvliet** [nextcloud/desktop#487]
Would be good to ensure our free space handling is clever enough to avoid issues.

Something like "stop all sync activity when free space gets below 100 mb" seems a good policy. Warn the user, don't start syncing again etc until it is resolved.

Pretty much any modern system gets in trouble at that point (SSD's don't deal well with being full over 95% for example) so if the user lets it get that far, they have to take action. The client shouldn't let it get to a point where a failed sync causes it to throw continuous errors or even do weird, potential data loss stuff.

**Comment by michaelstingl** [comment]
Related docs:
https://docs.nextcloud.com/desktop/2.3/advancedusage.html#low-disk-space

**Comment by jospoortvliet** [comment]
Ah, was already done, interesting, I still had this issue so it isn't working all too well... Let me see if I can reproduce it and create a new issue.
